config/roles.yml

```diff
@@ -284,13 +284,26 @@ ml_query_assistant_access:
 ml_read_access:
   reserved: true
   cluster_permissions:
+    - 'cluster:admin/opensearch/ml/config/get'
     - 'cluster:admin/opensearch/ml/connectors/get'
     - 'cluster:admin/opensearch/ml/connectors/search'
+    - 'cluster:admin/opensearch/ml/controllers/get'
```

Comment:
Don't forget to configure in security dashboard plugin https://github.com/opensearch-project/security-dashboards-plugin/pull/1182/files

Reply:
See https://github.com/opensearch-project/security-dashboards-plugin/pull/1838